petl/io/remotes.py

```diff
@@ -106,6 +106,9 @@ def _register_filesystems(only_available=False):
 def _register_filesystems_from(fsspec_registry, only_available):
     """Register each fsspec provider from this registry as remote source."""
     for protocol, spec in fsspec_registry.items():
+        if not isinstance(spec, dict):
+            continue
+
         missing_deps = "err" in spec
```

Comment:
This would still allow third-party filesystems to be registered in `petl`:

```suggestion
        missing_deps = isinstance(spec, dict) and "err" in spec
```